duckduckgo/iOS
# Fix network protection controller leak

Task/Issue URL: https://app.asana.com/0/414709148257752/1208331423616717/f
Tech Design URL:
CC:

**Description**:

Fix network protection controller leak



**Steps to test this PR**:
1. Open app in Instruments allocation tool.
2. Ensure there is only one instance of Network Controller.



**Definition of Done (Internal Only)**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
